ui/src/views/LocalRules.vue

```diff
@@ -835,6 +843,10 @@ export default {
       services: [],
       localServices: [],
       roles: [],
+      serviceOptions: {
+        placeholder: 'any',
```

Comment:
Remove placeholder.
No other dialog has it, so it could be misleading.

```suggestion
        placeholder: 'any',
```